custom/icds_reports/reports/disha.py

```diff
@@ -81,32 +81,32 @@ def _write_data_in_chunks(self, temp_path):
         #   }
         columns = self._get_columns()
         indicators = self._get_rows()
-        metadata_line = '{{'\
+        metadata_line = u'{{'\
```

Comment:
This is not necessary because `from __future__ import unicode_literals` is in this module.